account_operating_unit/views/account_payment_view.xml

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!--© 2016-17 Eficent Business and IT Consulting Services S.L.-->
+<!--© 2016 Serpent Consulting Services Pvt. Ltd.-->
+<!--License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).-->
+<odoo>
+    <record id="view_account_payment_tree" model="ir.ui.view">
+        <field name="name">account.payment.tree</field>
+        <field name="model">account.payment</field>
+        <field name="inherit_id" ref="account.view_account_payment_tree" />
+        <field name="arch" type="xml">
+            <field name="journal_id" position="after">
+                <field
+                    name="operating_unit_id"
+                    options="{'no_create': True}"
+                    groups="operating_unit.group_multi_operating_unit"
+                />
+            </field>
+        </field>
+    </record>
+    <record id="view_account_supplier_payment_tree" model="ir.ui.view">
```

Comment:
Supplier Payment Tree is show 2 columns for Operating Unit.
![Selection_118](https://user-images.githubusercontent.com/24691983/77915916-7af01e00-72c2-11ea-814f-22c1ac9067e4.png)